operations/_scripts/deploy/deploy.sh

```diff
@@ -53,6 +53,7 @@ docker run --rm --name bitops \
 -e SKIP_DEPLOY_HELM="${SKIP_DEPLOY_HELM}" \
 -e BITOPS_TERRAFORM_COMMAND="${TERRAFORM_COMMAND}" \
 -e TERRAFORM_DESTROY="${TERRAFORM_DESTROY}" \
+-e ANSIBLE_SKIP_DEPLOY="${ANSIBLE_SKIP_DEPLOY}" \
```

Comment:
We could make the stack_destroy: true set this to true, so that ansible doesn't run after destroy

Reply:
good idea!
- https://github.com/bitovi/github-actions-deploy-docker-to-ec2/issues/49
- https://github.com/bitovi/github-actions-deploy-stackstorm/issues/17